package.json

```diff
@@ -100,9 +92,10 @@
     "phantomjs": "1.9.18",
     "phantomjs-polyfill": "0.0.1",
     "proxyquire": "^1.0.0",
+    "rewire": "^2.3.4",
     "semver": "^5.0.3",
     "shelljs-nodecli": "~0.1.0",
-    "sinon": "^1.17.2",
+    "sinon": "1.17.2",
```

Comment:
Thank you for reviewing and sorry for the mess. Must have rushed something with the last merge.